packages/Search/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@paprika/search",
+  "version": "1.0.0",
```

Comment:
the initial version is better to < 1.0.0, otherwise the pre-release will publish `1.0.1-alpha0`, and the actual stable version will be at least 1.0.1